lib/EarlyFunctionAnalysis/FunctionMetadata.cpp

```diff
@@ -0,0 +1,382 @@
+/// \file ControlFlowGraph.cpp
```

Comment:
* Wrong name
* Any idea why github doesn't detect a moved file? It's hard to check for changes.